mermaid/src/worker.js

```diff
@@ -65,6 +67,42 @@ class Worker {
       }
     }
   }
+
+  updateConfig (task, config) {
+    for (const property in Object.fromEntries(config)) {
+      const propertyCamelCase = this.convertPropertyToCamelCase(property)
+      const value = this.getTypedValue(config.get(property))
+      _.set(task.mermaidConfig, propertyCamelCase, value)
```

Comment:
I would like to avoid adding lodash. I think we can use the spread operator and a tiny function to convert a path `er.useMaxWidth = true`  to an object: `{er: {useMaxWidth: true } }`

Reply:
I used lodash as it can safely handle nested elements if any. Its tested and popular and would be less buggy than whatever function I would come up with.

Why avoid lodash though?

Reply:
Fair enough, since Lodash has 0 dependency and the bundle size is relatively small, I willing to add it. Could you please use `lodash.set` instead of `lodash` (since we are only using `_.set`)?

Reply:
Lodash doesn't recommend per method packages anymore and will be removed in future versions as mentioned [here](https://lodash.com/per-method-packages)

Reply:
Oh I didn't know, in this case we should use:

```js
const set = require('lodash/set')
```

Right?

Reply:
We could, but since this is a server side code which won't be bundled, there is no benefit in trying to optimise the `require`. Moreover, `_` is pretty well understood to be `lodash` and imo, `_.set` is easier to understand.

Anyway, I will change it if you insist.

Reply:
I believe it will still load less code? I prefer to be explicit about what we are using. It prevents us from using Lodash where we don't actually need it. And, maybe one day, Node.js will have a built-in way of doing what Lodash does 😉